.travis.yml

```diff
@@ -34,8 +64,105 @@ install:
   # For now we disable it.
   - poetry config installer.parallel false
   - poetry install
+
+# Things to do before the script phase
 before_script:
-  - psql -U postgres -c 'create database nautobot;'
+  # If postgres: Create the database
+  - sh -c "if [ '$DB' = 'postgres' ]; then echo 'Creating PostgreSQL database...'; fi"
+  - sh -c "if [ '$DB' = 'postgres' ]; then psql -c 'CREATE DATABASE nautobot;' -U postgres; fi"
+  # If mysql: Upgrade/install MySQL 8.x, then create the database
+  - sh -c "if [ '$DB' = 'mysql' ]; then echo 'Upgrading MySQL to v8.x...'; fi"
+  - sh -c "if [ '$DB' = 'mysql' ]; then wget https://repo.mysql.com//mysql-apt-config_0.8.17-1_all.deb; fi"
+  - sh -c "if [ '$DB' = 'mysql' ]; then sudo dpkg -i mysql-apt-config_0.8.17-1_all.deb; fi"
```

Comment:
It's MySQL 5.7 by default. And yes, we have to list `mysql` in the base`services` because this install pattern upgrades MySQL from 5.7 to 8.x. (Source: https://akki.ca/blog/travis-ci-running-tests-on-mysql-8-0-x-on-ubuntu-xenial/)